File: SoftUni Lessons/Python Development/Python Basics April 2019/Past Exams and Problems/9 - 20 and 21 April 2019/03. Painting Eggs.py
size = input()
color = input()
count = int(input())

price = 0

if size == "Large":
    if color == "Red":
        price = count * 16
    elif color == "Green":
        price = count * 12
    elif color == "Yellow":
        price = count * 9
elif size == "Medium":
    if color == "Red":
        price = count * 13
    elif color == "Green":
        price = count * 9
    elif color == "Yellow":
        price = count * 7
elif size == "Small":
    if color == "Red":
        price = count * 9
    elif color == "Green":
        price = count * 8
    elif color == "Yellow":
        price = count * 5

price -= price * 0.35

print(f"{price:.2f} leva.")

File: SoftUni Lessons/JavaScript Development/JavaScript Basics March 2020/21. МИНАЛИ ИЗПИТИ/01. Online Exam - 20 and 21 April 2019/03. Painting Eggs.js
function paintingEggs(params) {
    let size = params.shift();
    let color = params.shift();
    let count = Number(params.shift());

    let prices = {
        'Large': {'Red': 16, 'Green': 12, 'Yellow': 9},
        'Medium': {'Red': 13, 'Green': 9, 'Yellow': 7},
        'Small': {'Red': 9, 'Green': 8, 'Yellow': 5}
    };

    console.log(`${((prices[size][color] * count) * 0.65).toFixed(2)} leva.`)
}

paintingEggs(['Large', 'Red', '7'])  // Expected Output: 72.80 leva.
paintingEggs(['Medium', 'Green', '5'])  // Expected Output: 29.25 leva.
paintingEggs(['Small', 'Yellow', '3'])  // Expected Output: 9.75 leva.
